wiki/Guides/Compressing_files/en.md

```diff
@@ -80,42 +92,74 @@ ffmpeg -i input -c:v libx264 -crf 20 -preset slower -profile:v high -vf scale=-1
+
+**NOTE:** Clicking `Cancel` on the metadata dialog will result in your audio file not being re-encoded.
+
 ### Using FFmpeg
 
-Paste the following command into your terminal and change the values as needed:
+*For instructions on installing FFmpeg, see: [Video/Using FFmpeg](/wiki/Guides/Compressing_files#using-ffmpeg)*
+
+After installing FFmpeg, open a terminal then use either of the following commands.
+
+To encode in the MP3 format, paste the following command into your terminal and change these values as needed:
 
 ```
 ffmpeg -i input -c:a libmp3lame -q:a 4 -vn -sn -map_metadata -1 -map_chapters -1 output.mp3
 ```
 
 - `-i input`: Your source file. If the file name contains spaces, wrap it around double quotes (`"`).
 - `-c:a libmp3lame`: Specify that the audio should be encoded using the LAME MP3 encoder.
-- `-q:a 4`: Use the same variable bitrate range as in the Audacity example, where a lower number means higher bitrate. If you want constant bitrate, you would instead use for instance `-b:a 128k` for a constant 128kbps bitrate.
+- `-q:a 4`: Use the same variable bit rate range as in the Audacity example, **where a lower number means higher bit rate**. If you want constant bit rate, you would instead use for instance `-b:a 128k` for a constant 128kbps bit rate.
 - `-vn -sn`: Remove video and subtitles if present.
 - `-map_metadata -1 -map_chapters -1`: Remove metadata and chapters if present.
 - `output.mp3`: Your output file. If the file name contains spaces, wrap it around double quotes (`"`).
 
+To encode in the OGG (Vorbis) format, paste the following command into your terminal and change these values as needed:
+
+```
+ffmpeg -i input -c:a libvorbis -q:a 5 -vn -sn -map_metadata -1 -map_chapters -1 output.ogg
+```
+
+- `-i input`: Your source file. If the file name contains spaces, wrap it around double quotes (`"`).
+- `-c:a libvorbis`: Specify that the audio should be encoded using the LAME MP3 encoder.
+- `-q:a 5`: Use the same variable bit rate range as in the Audacity example, **where a higher number means higher bit rate**. If you want constant bit rate, you would instead use for instance `-b:a 128k` for a constant 128kbps bit rate.
+- `-vn -sn`: Remove video and subtitles if present.
+- `-map_metadata -1 -map_chapters -1`: Remove metadata and chapters if present.
+- `output.ogg`: Your output file. If the file name contains spaces, wrap it around double quotes (`"`).
+
 ## Verification
 
-It is recommended to check the technical information of transcoded audio and video files to confirm that they have been processed as expected, and in a way that would meet bit rate and other requirements.
+It is recommended to check the technical information of re-encoded audio and video files to confirm that they have been processed as expected, and in a way that would meet bit rate and other requirements.
 
 Software such as [MediaInfo](https://mediaarea.net/en/MediaInfo) can be used to see such information.
```

Comment:
```suggestion
It is recommended to check the technical information of re-encoded audio and video files to confirm that they meet your expectations.
```

Reply:
this requires adding a link to https://mediaarea.net/en/MediaInfo after the `Using MediaInfo` heading:

```md
[MediaInfo](https://mediaarea.net/en/MediaInfo) is easy to use. <...>.
```